suricata/package.yaml

```diff
@@ -6,48 +6,1183 @@ author_icon: https://raw.githubusercontent.com/tenzir/library/main/author.svg
 package_icon: |
   https://raw.githubusercontent.com/tenzir/library/main/suricata/package.svg
 description: |
-  [Suricata](https://suricata.io) is an open-source network monitor and
+  [Suricata](https://suricata.io) is an open-source network monitoring and
   threat detection tool.
 
-  This package makes it easy to onboard EVE JSON logs that Suricata generates.
+  This package provides pipelines to onboard Suricata logs, as well as mapping
+  pipelines that translate Suricata EVE JSON to OCSF events.
 categories:
 - sources
+- mappings
 
 inputs:
-  eve-path:
+  eve-url:
     name: Suricata EVE JSON Output
     description: |
       The path configured in Suricata for writing EVE JSON.
     type: string
     default: /var/lib/suricata/eve.sock
 
 pipelines:
-  publish-suricata:
-    name: Publish Suricata EVE JSON
+  onboard-suricata:
+    name: Onboard Suricata EVE JSON
     description: |
       Publishes Suricata events read from the configured EVE JSON Output on the
       `suricata` topic.
     restart-on-error: true
     definition: |
-      load_file "{{ inputs.eve-path }}", follow=true
-      read_suricata
+      let $url = "{{ inputs.eve-url }}"
+      from_file $url, follow=true {
+        read_suricata
+      }
       publish "suricata"
```

Comment:
Did you test these changes? `from_file` doesn't have a `follow` option at the moment.

Reply:
Fixed.